app/main.py

```diff
@@ -4,4 +1,37 @@
+import os
+import requests
+
+
+API_URL = "http://api.weatherapi.com/v1/current.json"
+CITY = "Paris"
+
+
+def get_weather(api_key: str) -> None:
+    print(f"Performing request to Weather API for city {CITY}...")
+    params = {
+        "key": api_key,
+        "q": CITY
+    }
+    response = requests.get(API_URL, params=params)
+    if response.status_code == 200:
+        data = response.json()
+        location_time = data["location"]["localtime"]
+        temperature = data["current"]["temp_c"]
+        condition_text = data["current"]["condition"]["text"]
+        location = data["location"]["name"]
+        country = data["location"]["country"]
+        print(
+            f"{location}/{country} {location_time} Weather: "
+            f"{temperature} Celsius, "
+            f"{condition_text}"
+        )
+    else:
+        print("Error:", response.status_code, response.text)
```

Comment:
Code Quality Issue: The status code check and the data extraction from the response could be separated into two functions for better code organization and readability. One function could be responsible for making the request and checking the status code. If the status code is 200, this function could return the JSON data. The other function could take this data as an argument and extract the necessary information.